.rubocop.yml

```diff
@@ -34,6 +34,14 @@ Naming/MethodParameterName:
   Exclude:
     - 'app/components/arrow_component.rb'
 
+Naming/VariableNumber:
```

Comment:
I hate to exclude stuff here, but I don't see another way that isn't trying to circumvent rubocop's exclusion of indexes in `let` names on generic factories like this. At least this lets us keep track of where the offences are.